lib/Fencer/Server.hs

```diff
@@ -45,7 +45,7 @@ runServerWithPort (Port port) logger appState = do
     let options = Grpc.defaultServiceOptions
             { Grpc.serverHost = "0.0.0.0"
             , Grpc.serverPort = fromIntegral port
-              -- TODO: set the logger
+            , Grpc.logger     = \s -> Logger.info logger $ Logger.msg s
```

Comment:
I'd write `Grpc.logger     = Logger.info logger . Logger.msg`, but it's not important in any way.

Reply:
You're right! I attempted with something almost identical, but I got a type error and reverted to this. Definitely adopting your suggestion.